dm/dm/master/server.go

```diff
@@ -2580,3 +2580,179 @@ func (s *Server) sharedLogic(ctx context.Context, req interface{}, respPointer i
 	*errPointer = terror.ErrMasterRequestIsNotForwardToLeader
 	return true
 }
+
+func (s *Server) getSubTaskCfgs(taskName string, sources []string) (map[string]map[string]config.SubTaskConfig, error) {
```

Comment:
perfer add a func call `getStubTaskCfgByTaskAndSource` in`scheduler `

Reply:
I'll move it later.